src/os/config/settings.js

```diff
@@ -702,6 +702,15 @@ os.config.Settings.prototype.getLastReset = function() {
 };
 
 
+/**
+ * Retrieve the settings' peer info objects
+ *
+ * @return {Array<os.xt.PeerInfo>}
+ */
+os.config.Settings.prototype.getPeerInfo = function() {
```

Comment:
From an API perspective, do we want to expose the entire peer? If not, we should consider supporting the `opt_type` param for `getPeerInfo`.